src/transformation/visitors/break-continue.ts

```diff
@@ -11,7 +11,11 @@ export const transformBreakStatement: FunctionVisitor<ts.BreakStatement> = (brea
 };
 
 export const transformContinueStatement: FunctionVisitor<ts.ContinueStatement> = (statement, context) => {
-    if (context.luaTarget === LuaTarget.Universal || context.luaTarget === LuaTarget.Lua51) {
+    if (
+        context.luaTarget === LuaTarget.Universal ||
+        context.luaTarget === LuaTarget.Lua50 ||
+        context.luaTarget === LuaTarget.Lua51
+    ) {
         context.diagnostics.push(unsupportedForTarget(statement, "Continue statement", LuaTarget.Lua51));
```

Comment:
Same here, this diagnostic should get context.luaTarget instead of hardcoding Lua51